src/notice/dto/update-notice.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { CreateNoticeDto } from './create-notice.dto';
+
+export class UpdateNoticeDto extends PartialType(CreateNoticeDto) {}
```

Comment:
Update notice dto is same as create notice dto, but it is optinal.
so Add Partial Type